internal/controller/controllers/clusterdeployments_controller.go

```diff
@@ -183,6 +184,16 @@ func (r *ClusterDeploymentsReconciler) updateIfNeeded(ctx context.Context, clust
 		}
 	}
 
+	updateNonNullableString := func(new, old string, target *string, isInstallEnvUpdate bool) {
```

Comment:
@filanov a bit of a hack. need to find use `updateString()` instead. let me know what you think